homeassistant/components/mqtt/__init__.py

```diff
@@ -512,15 +519,13 @@ def websocket_mqtt_info(hass, connection, msg):
 )
 @websocket_api.async_response
 async def websocket_subscribe(
-    hass: HomeAssistant,
-    connection: websocket_api.ActiveConnection,
-    msg: dict[str, Any],
+    hass: HomeAssistant, connection: websocket_api.ActiveConnection, msg: dict[str, Any]
```

Comment:
I added type hints for `websocket_command` in the whole code base this week, so I think this due to a rebase.
I think code-style should be a separate PR.